proposed/internationalisation-meta.md

```diff
@@ -0,0 +1,39 @@
+# Internationalism Meta Document
+
+## 1. Summary
+
+Developers of components that are unattached to specific frameworks often find themselves in need of displaying a message to the user of the web application.  However, being as they are not a component of a specific framework - their options for making these messages localisable is limited.  In many cases, developers will make no attempt to pass these messages through a localisation layer.
```

Comment:
```suggestion
Developers of components that are not coupled to any specific framework often find themselves in need of displaying a message to the user of the web application. However, being as they are not a component of a specific framework - their options for making these messages localisable is limited. In many cases, developers will make no attempt to pass these messages through a localization layer.
```

Reply:
Actually, framework developers/users are interested in it since the more packages could be used in a framework the better.